test/fixtures/manage_rules_disabled/main.tf

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "tls" {
+  version = "~> 2.0"
+}
+
+resource "tls_private_key" "main" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+resource "local_file" "gce-keypair-pk" {
+  content  = tls_private_key.main.private_key_pem
+  filename = "${path.module}/sshkey"
+}
+
+data "google_compute_zones" "main" {
+  project = var.project_id
+  region  = var.region
+  status  = "UP"
+}
+
+#-------------------------#
+# Bastion Host
+#-------------------------#
+module "bastion" {
+  source = "../bastion"
+
+  network    = var.network
+  project_id = var.project_id
+  subnetwork = var.subnetwork
+  zone       = data.google_compute_zones.main.names[0]
+  key_suffix = "_manage_rules_disabled"
+}
+
+#-------------------------#
+# Forseti
+#-------------------------#
+module "forseti-manage-rules-disabled" {
```

Comment:
Thanks @choonchernlim! The build is failing because the Forseti server did not start. The default Forseti version right now is 2.24.0, which is probably hitting the idna issue. The [other examples use the master branch of Forseti](https://github.com/forseti-security/terraform-google-forseti/blob/master/test/fixtures/install_simple/variables.tf#L25), which I recommend here.

GCB errors:
```
null_resource.wait_for_server (remote-exec): Forseti was not ready after 720 seconds!
       
Error: error executing "/tmp/terraform_1299532757.sh": Process exited with status 1
```

Reply:
@gkowalski-google My understanding is other branches suffer the same `idna` error because that version isn't pinned. In my current production version, I actually hacked `modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl` and forced `pip` to pull `2.8` through `python3 -m pip install 'idna==2.8'` before `setup.py` runs since there's no tagged release from Forseti Security yet.

Regarding that error, it seems I need to increase the wait time, maybe to 900 seconds or higher to allow the provisioning to complete: https://github.com/choonchernlim/terraform-google-forseti/blob/_feature-empty-rules-dir/test/fixtures/manage_rules_disabled/scripts/wait-for-forseti.sh#L21 ... but, I will need to hold off until the next version of Forseti Security is released.

Reply:
My bad. I didn't realized my PR got merged into Forseti Security master. Sorry. 

I have been working with too many copies of Forseti. LOL. I increased the timeout, and let's see if that fixes the problem. Thanks.

Reply:
Haha okay thanks, I don't think we need the increased wait timeout but that's ok. Using the master version of Forseti should resolve it. Thanks for all of your recent contributions!

Reply:
Thank you for answering all my questions. I totally appreciate it! I'm trying to get the scripts ready to provision Forseti in production in early March. So, I can definitely help in terms of testing and fixing simple problems.